_includes/pbc/all/install-features/202212.0/install-the-dynamic-multiple-stores.md

```diff
@@ -2518,6 +2518,95 @@ Also you need to change domain name for all endpoints in EU region.
 Please, check `deploy.dev.dynamic-store.yml` file for more details.
 
 
+## Data migration 
+
+You don't need to migrate data in the database for the dynamic store feature. Only changes in the search and storage engine are necessary.
+Propel migration will be executed automatically and added new tables for dynamic store feature and populate new tables with data via data import.
+For correct work of dynamic store feature you need to add new data structure for search and storage engines.
+
+
+First of all, you need to truncate all data from search and storage engines. 
+Please see below for more details in the section **"Truncate data from search and storage engines"**.
+Next step is run sync:data command for populate new data structure for search and storage engines.
+
+```bash
+vendor/bin/console sync:data
+```
+
+All data will be populated in search and storage engines.
+
+{% info_block warningBox "Verification" %}
+
+
+### Truncate data from search and storage engines
+
+#### Storage engine
+
+Since Dynamic Store feature uses Redis as a key-value store for each region instead of store. If using same Redis instance for region, you need to clear all data from Redis for existing stores.
+The data is stored in the following format: `kv:{resource-type}:{store}:{locale}:{key}`.
+ 
+ For remove data from Redis you can use `redis-cli` command:
+ 
+ ```bash
+ redis-cli KEYS "*:xxx:*" | xargs redis-cli DEL
+```
+
+{% info_block warningBox  %}
+Note that the key name can be different, depending on the configuration of the project.
+Below is a list of keys, taking into account the default configuration out of the box.
+{% endinfo_block %}
+
+
+{% info_block warningBox "Verification" %}
+Example list of keys where `xxx` - is a store name: 
+
+- `kv:availability:xxx:*`
+- `kv:price_product_abstract:xxx:*`
+- `kv:price_product_abstract_merchant_relationship:xxx:*`
+- `kv:price_product_concrete:xxx:*`
+- `kv:price_product_concrete_merchant_relationship:xxx:*`
+- `kv:product_abstract:xxx:*`
+- `kv:product_abstract_category:xxx:*`
+- `kv:product_abstract_option:xxx:*`
+- `kv:product_abstract_relation:xxx:*`
+- `kv:product_concrete_measurement_unit:xxx:*`
+- `kv:product_concrete_product_offer_price:xxx:*`
+- `kv:product_concrete_product_offers:xxx:*`
+- `kv:product_label_dictionary:xxx:*`
+- `kv:product_offer:xxx:*`
+- `kv:product_offer_availability:xxx:*`
+- `kv:category_node:xxx:*`
+- `kv:category_tree:xxx:*`
+- `kv:cms_block:xxx:*`
+- `kv:cms_page:xxx:*`
+- `kv:merchant:xxx:*`
+- `kv:price_product_abstract_merchant_relationship:xxx:*`
+- `kv:store:xxx`
+
+{% endinfo_block %}
+
+But recomedation flash all data from Redis for instance and populate it again.
```

Comment:
```suggestion
It's recommended to flash all data from Redis by instance and populate them again.
```

Reply:
@AlexSlawinski Please review again this part.